auth-api/src/auth_api/services/org.py

```diff
@@ -815,26 +815,30 @@ def approve_or_reject(org_id: int, is_approved: bool, origin_url: str = None):
 
     @staticmethod
     @user_context
-    def send_staff_review_account_reminder(org_id, origin_url, **kwargs):
+    def send_staff_review_account_reminder(org_id, origin_url, task_relationship_type=TaskRelationshipType.ORG.value,
+                                           **kwargs):
         """Send staff review account reminder notification."""
         current_app.logger.debug('<send_staff_review_account_reminder')
         recipient = current_app.config.get('STAFF_ADMIN_EMAIL')
-        context_path = f'review-account/{org_id}'
+        # Get task id that is related with the task. Task Relationship Type can be ORG or PRODUCT.
+        task = TaskModel.find_by_task_relationship_id(task_relationship_type=task_relationship_type,
+                                                      relationship_id=org_id)
+        context_path = f'review-account/{task.id}'
         app_url = '{}/{}'.format(origin_url, current_app.config.get('AUTH_WEB_TOKEN_CONFIRM_PATH'))
         review_url = '{}/{}'.format(app_url, context_path)
         user_from_context: UserContext = kwargs['user_context']
         first_name = user_from_context.first_name
         last_name = user_from_context.last_name
 
         data = {
-            'accountId': org_id,
+            'accountId': task.id,
```

Comment:
if it's not accountId, then ideally we should rename it, as passing task id as account id is not good for maintenance

Reply:
yeah. Removed it as we do not need account id for this particular template. Context url where task id is appended is only required